Create hlces-hamburg.csl

eu-genia opened the pull request:
A style used in the numerous publications of the Hiob Ludolf Centre of Ethiopian Studies (HLCES), University of Hamburg: the journals Aethiopica and the Comparative Oriental Manuscript Studies Bulletin, the series Aethiopistische Forschungen, the series Supplements to Aethiopica.

zuphilip commented on hlces-hamburg.csl:
```diff
@@ -0,0 +1,339 @@
+              <date-part name="month" form="long" prefix=" "/>
+            </date>
+            <group prefix=" (" suffix=")">
+              <text term=" " suffix=" "/>
```
That is not valid. What do you want here? Maybe something like
```<text term="retrieved" suffix=" "/>```

zuphilip commented on hlces-hamburg.csl:
```diff
@@ -0,0 +1,339 @@
+    <title-short>HLCES</title-short>
+    <id>http://www.zotero.org/styles/hlces-hamburg</id>
+    <link href="http://www.zotero.org/styles/hlces-hamburg" rel="self"/>
+    <link rel="template" href="http://www1.uni-hamburg.de/COMST/bulletin.html"/>
```
That is actually you documentation and therefore you should use `rel="documentation"`.

Did you base your style on another style? Which style did you use as a template?

zuphilip commented:
These two comments will help to make the automated checks passing. Afterwards someone will look at it closer. I guess that the we should write the name out, i.e. `Hiob Ludolf Centre of Ethiopian Studies` and use the abbreviation in the special field for it. Moreover, it is possible to create afterwards the styles for the journals as dependent styles which can then be found individually and all of them using the exact same style.

eu-genia commented:
Thank you so much for your help, I believe to have addressed the  
issues signalled.

A question I have not yet found a solution to: in our style we  
actually should not have a dot after the plural like eds. (editor  
label) - our style should be eds - is there any way to adjust this?

Thank you

Eugenia


Quoting Philipp Zumstein <notifications@github.com>:

> These two comments will help to make the automated checks passing.  
> Afterwards someone will look at it closer. I guess that the we  
> should write the name out, i.e. `Hiob Ludolf Centre of Ethiopian  
> Studies` and use the abbreviation in the special field for it.  
> Moreover, it is possible to create afterwards the styles for the  
> journals as dependent styles which can then be found individually  
> and all of them using the exact same style.
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1977#issuecomment-200530896



-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian Studies
Asien Afrika Institut
Universit�t Hamburg
Alsterterrasse 1
20354 Hamburg

eugenia.sokolinski@uni-hamburg.de

http://www1.uni-hamburg.de/ethiostudies
http://www.traces.uni-hamburg.de

zuphilip commented:
<blockquote>in our style we actually should not have a dot after the plural like eds. (editor label) - our style should be eds - is there any way to adjust this?</blockquote>
Yes, you can set the [`strip-periods`](http://docs.citationstyles.org/en/stable/specification.html#strip-periods) option at the relevant label.

eu-genia commented:
The option seems to strip periods from the label in both sing. and  
plural - we have contextual use and the period should be stripped in  
the plural only (ed. but eds)

Quoting Philipp Zumstein <notifications@github.com>:

> <blockquote>in our style we actually should not have a dot after the  
> plural like eds. (editor label) - our style should be eds - is there  
> any way to adjust this?</blockquote>
> Yes, you can set the  
> [`strip-periods`](http://docs.citationstyles.org/en/stable/specification.html#strip-periods) option at the relevant  
> label.
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1977#issuecomment-200780794



-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian Studies
Asien Afrika Institut
Universit�t Hamburg
Alsterterrasse 1
20354 Hamburg

eugenia.sokolinski@uni-hamburg.de

http://www1.uni-hamburg.de/ethiostudies
http://www.traces.uni-hamburg.de

zuphilip commented:
Ah, I see. Then you can try to redefine the term in your locale, i.e.
```
<term name="editor" form="short">
   <single>ed.</single>
   <multiple>eds</multiple>
</term>
```
(Is this really a useful convention or maybe it just slipped through some official documentation of the style?)

eu-genia commented:
This was an editorial decision of our department head (admissibile  
according to the latest Oxford manual of style).
Thanks, solved!

Quoting Philipp Zumstein <notifications@github.com>:

> Ah, I see. Then you can try to redefine the term in your locale, i.e.
> ```
> <term name="editor" form="short">
>    <single>ed.</single>
>    <multiple>eds</multiple>
> </term>
> ```
> (Is this really a useful convention or maybe it just slipped through  
> some official documentation of the style?)
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1977#issuecomment-200796597



-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian Studies
Asien Afrika Institut
Universit�t Hamburg
Alsterterrasse 1
20354 Hamburg

eugenia.sokolinski@uni-hamburg.de

http://www1.uni-hamburg.de/ethiostudies
http://www.traces.uni-hamburg.de

rmzelle commented:
Unless you plan to expand the contents of https://github.com/eu-genia/citation, it might be handier to have a direct "documentation" link to one of the journal submission guidelines, e.g. "http://www1.uni-hamburg.de/COMST/bulletin.html".

You could also quickly run your style through http://formatter.citationstyles.org/ (copy/paste, click button) to correct the indentation. Otherwise everything looks good.

adam3smith commented:
@zuphilip -- FYI: ed. and eds is very common in British English and does have a good rationale: the "s" in "editors" (and other plurals) is at the end, so there shouldn't be a period behind it suggesting the word gets truncated. It's not how we think about abbreviations in in American English, but it's sufficiently standard in the UK that we could arguably even code this into the `en-GB` locale (and I think we might for something else?).

rmzelle commented:
Thanks!

Were you already planning to create dependent CSL styles for the individual journals?

eu-genia commented:
Just a question first to be sure, if I may - if we have to change  
something in the format - do I understand it right that it shall  
suffice to change the main style and the dependent ones shall be  
updated automatically?
Thanks!

Quoting "Rintze M. Zelle" <notifications@github.com>:

> Thanks!
>
> Were you already planning to create dependent CSL styles for the  
> individual journals?
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1977#issuecomment-200979362



-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian Studies
Asien Afrika Institut
Universit�t Hamburg
Alsterterrasse 1
20354 Hamburg

eugenia.sokolinski@uni-hamburg.de

http://www1.uni-hamburg.de/ethiostudies
http://www.traces.uni-hamburg.de

adam3smith commented:
that's correct, yes.

rmzelle commented:
Let us know if you need any help creating the dependents. (we have a tool to generate dependents for larger journal collections, that if you only have four that's not really worth it)

eu-genia commented:
I have tried but something is wrong so I do need help (I have pulled a  
new request for a dependent style - comparative oriental manuscript  
studies bulletin, it is the same but I renamed it and added a link to  
the parent style - is anything else needed?)


Quoting "Rintze M. Zelle" <notifications@github.com>:

> Let us know if you need any help creating the dependents. (we have a  
> tool to generate dependents for larger journal collections, that if  
> you only have four that's not really worth it)
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1977#issuecomment-201085355



-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian Studies
Asien Afrika Institut
Universit�t Hamburg
Alsterterrasse 1
20354 Hamburg

eugenia.sokolinski@uni-hamburg.de

http://www1.uni-hamburg.de/ethiostudies
http://www.traces.uni-hamburg.de
